### PR TITLE
INSERT rules-tested transaction items

### PR DESCRIPTION
**Pull request by oozywaters**

### CODE REVIEW

File: services/transact-faas/package.json [hunk 1]
```diff
@@ -12,6 +12,8 @@
     "deploy:dev": "yarn run zip && yarn run update:faas:dev"
   },
   "dependencies": {
+    "aws-sdk": "^2.395.0",
```
Comment (mxfactorial) [inline services/transact-faas/package.json]: Unless version > 2.290.0 required, no need to add `aws-sdk` as local dependency: https://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html

File: services/graphql-faas/package.json [hunk 2]
```diff
@@ -20,7 +20,6 @@
     "graphql": "^14.0.2"
   },
   "devDependencies": {
-    "jest": "^24.0.0",
```
Comment (mxfactorial) [inline services/graphql-faas/package.json:23]: breaks [jest dependency](https://github.com/systemaccounting/mxfactorial/blob/611348778a66f9cf98ec3571d20fa9445fbbb8ff/services/graphql-faas/package.json#L7-L9)

**Comment by oozywaters** [comment]
Ready for review.

**Comment by mxfactorial** [comment]
`tests` directories missing from [transact](https://github.com/systemaccounting/mxfactorial/tree/insert-transactions/services/transact-faas) and [rules](https://github.com/systemaccounting/mxfactorial/tree/insert-transactions/services/rules-faas). for example, exported [addTransactions.js](https://github.com/systemaccounting/mxfactorial/blob/95e7af9db7ea1585f0914ce5406988010df8b020/services/transact-faas/src/addTransactions.js) easily testable

please:
1. add `tests` directories rather than adjacent test files for convenient exclusion before deployment
1. add `jest` and tests
1. remove unused packages, e.g. mysql2
1. add `yarn test && zip -r ...` to `build.js:5` in [transact](https://github.com/systemaccounting/mxfactorial/blob/95e7af9db7ea1585f0914ce5406988010df8b020/services/transact-faas/build.sh#L5) and [rules](https://github.com/systemaccounting/mxfactorial/blob/95e7af9db7ea1585f0914ce5406988010df8b020/services/rules-faas/build.sh#L5) to fail archive if test fails

increment estimation in https://github.com/systemaccounting/mxfactorial/issues/91 if underestimated

supply line range links to automated tests of these requirements in https://github.com/systemaccounting/mxfactorial/issues/91 when complete:

**/transact service tests transactions items against /rules**
4. /transact service tests itemsUnderTestArray for equality with itemsStandardArray

**INSERT into transactions table**
2. /transact service INSERTs itemsUnderTestArray into transactions table

**unequal transaction items**
2. /transact service returns `required items missing` error to client
3. /transact service does **NOT** INSERT itemsUnderTestArray into transactions table

**Comment by oozywaters** [comment]
> remove unused packages, e.g. mysql2

mysql2 is required by sequelize when using mysql dialect

**Pull request merged**